wcfsetup/install/files/acp/templates/reactionTypeList.tpl

```diff
@@ -46,6 +47,7 @@
 						
 						<span class="statusDisplay sortableButtonContainer">
 							<span class="icon icon16 fa-arrows sortableNodeHandle"></span>
+							<span class="jsOnly icon icon16 fa-{if $reactionType->isAssignable}check-{/if}square-o jsToggleButton jsTooltip pointer" title="{lang}wcf.global.button.{if !$reactionType->isAssignable}enable{else}disable{/if}{/lang}" data-object-id="{@$reactionType->reactionTypeID}"></span>
```

Comment:
The description text does not fit. We are talking about "disable" or "activate", but we should be talking about "assignable" or "not assignable", so that the user knows what the button does, just because in the detail editing we are not talking about disable/enable.

Reply:
I'm unsure about the wording
`Verwendbar machen` / `Set assignable` and `Unverwendbar machen` / `Set nonassignable` sounds strange.

Reply:
Please use `Verwendung erlauben` and `Verwendung verbieten` and `Allow use` and `Disallow use` in english. I will adjust the detailed page soon to `Verwendung erlauben` and `Allow use` (or, if you like, you can also change the language variable with the pull request, too).

Reply:
I hope I got everything done.